airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlCdcStateHandler.java

```diff
@@ -12,6 +12,7 @@
 import io.airbyte.protocol.models.v0.AirbyteMessage;
 import io.airbyte.protocol.models.v0.AirbyteMessage.Type;
 import io.airbyte.protocol.models.v0.AirbyteStateMessage;
+import io.debezium.engine.ChangeEvent;
```

Comment:
why do we need this? i think this is not required

Reply:
We don't.  I'm not sure how that got in there.  I suspect this is another rebase issue.  Will remove